### DISCUSSION TITLE
Filtering by clicking on a persons icon is broken

### DISCUSSION THREAD

**Discussion by wilber92** [community/community#11942] [Projects and Issues]
This is what it filters when i click on a person:
milestone:"Sprint 6 - 11th Mar assignee:benjaminhugecalf 

What it should filter:
milestone:"Sprint 6 - 11th Mar" assignee:benjaminhugecalf 

Its missing a " before the assignee

**Comment by andersc** [comment]
Just want to confirm this issue. Still there 23-Feb 2022.
